docs/resources/cnad_advanced_policy.md

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "Cloud Native Anti-DDoS Advanced"
+---
+
+# huaweicloud_cnad_advanced_policy
+
+Manages a CNAD advanced policy resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_cnad_advanced_instances" "test" {}
+
+resource "huaweicloud_cnad_advanced_policy" "test" {
+  instance_id = huaweicloud_cnad_advanced_instances.test.instances[0].instance_id
+  name        = "test-policy"
+  threshold   = 100
+  udp         = "block"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `instance_id` - (Required, String, ForceNew) Specifies the CNAD advanced instance ID.
+  You can find it through data source `huaweicloud_cnad_advanced_instances`. If you have no instances, please
+  buy one first.
```

Comment:
`If you have no instances, please buy one first.` --- it's unnecessary to add it.